components/inputs/test/input-time.test.js

```diff
@@ -0,0 +1,130 @@
+import '../input-time.js';
+import { aTimeout, expect, fixture, oneEvent } from '@open-wc/testing';
+
+const basicFixture = '<d2l-input-time label="label text"></d2l-input-time>';
+
+describe('d2l-input-time', () => {
+
+	let elem;
+
+	beforeEach(async() => {
+		elem = await fixture(basicFixture);
+	});
+
+	function getInput() {
```

Comment:
Can you move this to the top of the file to stay consistent with what I've been doing elsewhere? It'll need to take `elem` as a parameter.